### ISSUE TITLE
After upgrading to rollup v0.46 this plugin causes rollup not exiting normally

### ISSUE THREAD

**Issue by ClassicOldSong** [4lejandrito/rollup-plugin-browsersync#1]
When pressing `Ctrl+C`, rollup cannot stop automatically. After a few attempts it seems to be closed but actually it still has a background process running. Browsersync is still watching and serving and reloading the page. Hope you'll have it fixed, thanks!

**Comment by 4lejandrito** [comment]
Hi @ClassicOldSong,

It works for me. The module is already taking this into account (see [this bit of code](https://github.com/4lejandrito/rollup-plugin-browsersync/blob/master/index.js#L7-L11)).

In order to be able to reproduce the issue, could you please let me know which **version of node** and which **operating system** are you running it from?

Thanks!

**Comment by ClassicOldSong** [comment]
Screenfetch:

```

                 -/+:.          yukino@YukinoPrivate
                :++++.          OS: 64bit Mac OS X 10.13 17A405
               /+++/.           Kernel: x86_64 Darwin 17.0.0
       .:-::- .+/:-``.::-       Uptime: 1d 40m
    .:/++++++/::::/++++++/:`    Packages: 72
  .:///////////////////////:`   Shell: bash 3.2.57
  ////////////////////////`     Resolution: 2880x1800
 -+++++++++++++++++++++++`      DE: Aqua
 /++++++++++++++++++++++/       WM: Quartz Compositor
 /sssssssssssssssssssssss.      WM Theme: Blue
 :ssssssssssssssssssssssss-     Font: LiterationMonoPowerline 12
  osssssssssssssssssssssssso/`  CPU: Intel Core i7-6700HQ @ 2.60GHz
  `syyyyyyyyyyyyyyyyyyyyyyyy+`  GPU: Intel HD Graphics 530 / AMD Radeon Pro 450
   `ossssssssssssssssssssss/    RAM: 9170MiB / 16384MiB
     :ooooooooooooooooooo+.
      `:+oo+/:-..-:/+o+/-
```

```
❯ node -v
v8.7.0
```

Steps to reproduce:

```
git clone https://github.com/TheNeuronProject/ef.js.git
cd ef.js
git checkout dev
npm install
npm run dev
```

Wait a while and try `ctrl+c`.

**Comment by 4lejandrito** [comment]
Hi @ClassicOldSong,

I have fixed the problem and published a new version (0.2.5). Please reopen if you think it is not fixed.

Thanks!

**Comment by ClassicOldSong** [comment]
Seems fixed, thanks!
